### PR TITLE
Finished the Board class, and updated the TurnOrder class.

### PR DESCRIPTION
**Pull request by Peter-Scheelke**
The Board's code coverage is above 95%, and TurnOrder is at 100%. Both pass Stylecop.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/sh4ne/Scrabble_CS3450/pull/48" target="_blank">ReviewNinja

**Comment by Peter-Scheelke** [comment]
Obligatory :+1:

File: Scrabble/Game Objects/Board.cs [hunk 1]
```diff
@@ -1,23 +1,857 @@
...
+        }
+
+        /// <summary>
+        /// Tells whether a <see cref="Play"/> can legally be placed onto the board.
+        /// </summary>
+        /// <param name="play">The <see cref="Play"/> being checked.</param>
+        /// <returns>True if play can be added to the board. Returns false otherwise.</returns>
+        public bool PlayIsValid(Play play)
+        {
+            if (play.GetParallelListLength() == 0)
```
Comment (johnsonjo4531) [inline Scrabble/Game Objects/Board.cs:67]: Parallel list length should never be zero it would throw an EmptyParrallelList() error.

**Comment by sh4ne** [comment]
I didn't see anything that needs to be changed as I read through the code. :+1:

File: Scrabble/Game Objects/Board.cs (hunk 1 above)
Comment (Peter-Scheelke) [inline Scrabble/Game Objects/Board.cs:67]: Hmm, it was letting me do it. I must have an older version of Play or something. Oh well, it shouldn't matter, and we might as well have redundancy.

**Comment by Peter-Scheelke** [comment]
I made a few changes. Plays can be scored without removing score multipliers, there is now a 50 point bonus for using 7 letter tiles, and the most recent play can be removed from the board.

**Comment by johnsonjo4531** [comment]
:+1: Sadly, I was actually working on the board class at the same time as you. I didn't know you were working on it. I'll just scrap mine, because I wasn't finished, but I really like your code it looks good. 

I was also wondering whether the board should handle saying what all the words are that are created by that play so that it can be checked against the dictionary at some point. 

By the way I'm not sure what the best approach is for making sure we don't work on the same thing. I didn't check if there was an issue for finishing the board class if there was than it was totally my fault for not checking. It might be one possibility, to create an issue and assign ourselves to it, so it is less likely to happen again. I only started on it over thanksgiving, because Shane told me to work on adding the play to the board. It's not too big of a deal, but I think if we followed a set way of being assigned issues we might be able to stop this from happening again.

**Comment by Peter-Scheelke** [comment]
It was an issue, but it got finished for some reason. I worked on it because I needed it done to do the game world (the board took a bit longer than I expected).

That's a good point. The board should probably be a method that returns a list of the words made by a play. Then the game world could check each of them against the dictionary in the case of a challenge.

**Comment by sh4ne** [comment]
Are we all good on this pull? And we will have to discuss what happened and how we will avoid it in the future tomorrow.

**Comment by sh4ne** [comment]
I'm going to merge the changes, if something needs to be changed, make a new pull request.

**Pull request merged**